.github/workflows/release.yml

```diff
@@ -33,14 +33,16 @@ jobs:
 
       - name: Commit the app.conf changes
         run: |
-          git commit -am "chore: update app.conf version to ${{ steps.tag.outputs.tag }}"
+          git add defaults/app.conf
+          git commit -m "chore: update app.conf version to ${{ steps.tag.outputs.tag }}"
 
       - run: slim package -o target/ .
 
       - run: cp target/splunk_heroku_log_drain-${{ steps.tag.outputs.tag }}.tar.gz splunk_heroku_log_drain-latest.tar.gz
 
       - name: Commit the splunk_heroku_log_drain-latest.tar.gz changes
         run: |
-          git commit -am "chore: update splunk_heroku_log_drain-latest.tar.gz to version ${{ steps.tag.outputs.tag }}"
```

Comment:
Here `-a` only adds changed files, but not new files. Being more explicit seems better anyway.